index.html

```diff
@@ -175,12 +176,13 @@ <h1 id="title">Publishing Working Group Charter</h1>
 
             <section id="scope" class="scope">
                 <h2>Scope</h2>
+                
                 <p>
-                    For the purpose of this document, a Web Publication (WP) is a collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies.
-                    A Web Publication is not just a collection of links. The act of publishing involves obtaining resources and organizing them into a publication, which must be “manifested” by having files on a Web server.
-                    Thus the publisher provides an <a href="https://tools.ietf.org/html/rfc6454">origin</a> for the WP, and a URL that can uniquely identify that <a href="https://archive.ifla.org/VII/s13/frbr/frbr1.htm#3.2">(FRBR) manifestation</a>.
-                    A Web Publication must provide a number of features whose detailed specification is in the scope of this Working Group.
-                    While some of the detailed requirements have already been <a href="https://www.w3.org/TR/pwp-ucr/">documented elsewhere</a>, the most important and high-level characteristics that must be translated into specifications are:
+                    <strong>Definition: Web Publication (WP):</strong> A collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies is a collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies.
+                    <!--A Web Publication is not just a collection of links.--> The act of publishing involves obtaining resources and organizing them into a publication, which must be “manifested” by having files on a Web server.
+                    Thus the publisher provides an <a href="https://tools.ietf.org/html/rfc6454">origin</a> for the WP, and a URL that can uniquely identify that <a href="https://archive.ifla.org/VII/s13/frbr/frbr1.htm#3.2">(FRBR) (Functional Requirements for Bibliographic Records) manifestation</a>.

```

Comment:
Minor nit, but couldn't this just be: <abbr title="Functional Requirements for Bibliographic Records">FRBR</abbr>

A paren following a paren is strange. Alternatively, maybe wrap the title inside so they're more explicitly linked (FRBR (...)), or drop the parentheses around FRBR.